### PR TITLE
COMT

### PR DESCRIPTION
**Pull request by bnfish**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
+"use strict"
 var express = require('express');
 var router = express.Router();
+var pgp = require('pg-promise')();
+var pq = require('pg-promise').ParameterizedQuery;
+
+const connectionstring = `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_HOST}:${process.env.POSTGRES_PORT}/${process.env.POSTGRES_DB}`;
+const db = pgp(connectionstring);
```
Comment (lukecampbell) [inline routes/index.js]: Please don't make a connection at the root scope of the module. If this connection is slow or fails the module won't load. The guide for postgres in nodeJS recommends that the connection should be made in the response handler.

Comment (lukecampbell) [inline routes/index.js]: See the section under _Create_

http://mherman.org/blog/2015/02/12/postgresql-and-nodejs/

**Comment by bnfish** [comment]
appreciate the guidance, @lukecampbell; only reason i did that was because i was receiving the following warning every time i navigated to another comt page:

<kbd></kbd>

**Comment by lukecampbell** [comment]
It looks like pgpromise operates a bit differently than the postgres library in the post I mentioned. According to the API it defers an actual postgres connection until a request is made so how you had it before was more correct than creating duplicate objects. 

I'll take responsibility for merging this, I'm going to put the connection object in a different node module and export the connection.

**Pull request merged**